socorro/external/rabbitmq/test_rabbitmq.py

```diff
@@ -1,6 +1,6 @@
 import pika
```

Comment:
This is a strangely named file. The `test_` convention screams of unit testing. When in fact, it's a script. 
Perhaps `rabbitmq-test.py`? The use of a hyphen instead of an underscore quickly indicates that it's not a module you're suppose to ever import from python.